Armadillo/src/main/java/com/scribd/armadillo/Reducer.kt

```diff
@@ -175,14 +175,29 @@ internal object Reducer {
                             isSeeking = action.isSeeking,
                             seekTarget = action.seekPositionTarget)
                 } else MediaControlState()
-
+                val progress = if (action.seekPositionTarget != null) {
+                    playbackInfo.progress.copy(
+                        positionInDuration = action.seekPositionTarget,
```

Comment:
I guess the question to ask is "Should we consider the user at the new position when they _start_ a discontinuous action like seeking or jumping?" I feel like the answer should be "no", but I'm willing to be convinced it makes sense.

How does the scrubber/UI get updated if you jump while paused? Can we hitch the progress update into whatever makes that update?